extensions/datastores/foundationdb/src/main/java/org/locationtech/geowave/datastore/foundationdb/config/FoundationDBOptions.java

```diff
@@ -0,0 +1,18 @@
+package org.locationtech.geowave.datastore.foundationdb.config;
+
+import org.locationtech.geowave.core.store.DataStoreOptions;
+import org.locationtech.geowave.core.store.StoreFactoryFamilySpi;
+import org.locationtech.geowave.core.store.StoreFactoryOptions;
+
+// TODO: implement this
+public class FoundationDBOptions extends StoreFactoryOptions {
+  @Override
+  public StoreFactoryFamilySpi getStoreFactory() {
+    return null;
```

Comment:
you should be able to return the FDB store factory family here at least

Reply:
Good point.